items.md

```diff
@@ -137,6 +137,172 @@ Example: <https://dspace7.4science.it/dspace-spring-rest/#https://dspace7.4scien
 
 It returns the collection where the item belong to
 
+### Mapping Collections
+**GET /api/core/items/<:uuid>/mappingCollections**
+
+Example:
+```json
+    "mappingCollections":
+    [
+      {
+        "id": "16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9",
+        "uuid": "16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9",
+        "name": "Bachelor theses",
+        "handle": "123456789/5287",
+        "metadata": [
+          {
+            "key": "dc.description.abstract",
+            "value": "",
+            "language": null
+          },
+          {
+            "key": "dc.title",
+            "value": "Bachelor theses",
+            "language": null
+          }
+        ],
+        "type": "collection",
+        "_links": {
+          "license": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9/license"
+          },
+          "exportToZip": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9/exportToZip"
+          },
+          "defaultAccessConditions": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9/defaultAccessConditions"
+          },
+          "logo": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9/logo"
+          },
+          "self": {
+            "href": "https://dspace7-internal.atmire.com/rest/api/core/collections/16a4b65b-3b3f-4ef5-8058-ef6f5a653ef9"
```

Comment:
I suggest to only use the official REST and UI demos for the example. This will make also easier to update later all the reference to a dspace.org site

Reply:
We will update these later. Having them in here currently has the advantage that the URIs actually resolve (apart from the fact that mapping has been renamed to mapped in the latest update of the rest contract). This can be easily updated once the API is also live